projects/plugins/jetpack/extensions/store/membership-products/actions.js

```diff
@@ -32,7 +32,8 @@ export const saveProduct =
 		product,
 		productType = PRODUCT_TYPE_PAYMENT_PLAN,
 		setSelectedProductId = () => {},
-		callback = () => {}
+		callback = () => {},
+		silentProductCreation = false
```

Comment:
What do you think about giving this a more descriptive name like `shouldDisplayProductCreationNotice`?